catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -117,6 +124,17 @@ public ContentDirectoryMonitor(CamelContext camelContext, int maxRetries,
         this.maxRetries = maxRetries;
         this.delayBetweenRetries = delayBetweenRetries;
         this.configurationExecutor = configurationExecutor;
+        setBlacklist();
+    }
+
+    private void setBlacklist() {
+        String badFileProperty = System.getProperty("bad.files");
+        String badFileExtensionProperty = System.getProperty("bad.file.extensions");
+        badFiles = StringUtils.isNotEmpty(badFileProperty) ?
+                Arrays.asList(badFileProperty.split(",")) :
```

Comment:
I'm not sure it's necessary, as this is property is set by reading https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/system.properties#L118